examples/run/cuda/CMakeLists.txt

```diff
@@ -29,10 +29,10 @@ target_link_libraries( traccc_examples_cuda
    PUBLIC CUDA::cudart vecmem::core vecmem::cuda traccc::core
           traccc::device_common traccc::cuda )
 
-traccc_add_executable( throughput_st_cuda "throughput_st.cpp"
+traccc_add_executable( throughput_st_cuda "throughput_st.cpp" "inst_nvtx.cpp"
    LINK_LIBRARIES vecmem::core vecmem::cuda traccc::io traccc::performance
                   traccc::core traccc::device_common traccc::cuda
-                  traccc::options traccc_examples_cuda )
+                  traccc::options traccc_examples_cuda ${CMAKE_DL_LIBS})
```

Comment:
This worries me a little (a lot). Why is this necessary? :worried:

Reply:
The problem here is that the code compiles with no problem on my local machine. However, I get an error saying:
`undefined reference to symbol 'dlclose@@GLIBC_2.2.5`
when compiling on github. I found the fix for this on this link: https://stackoverflow.com/questions/56841870/how-to-fix-undefined-reference-to-symbol-dlcloseglibc-2-2-5-from-glad-c

Reply:
Ah okay yeah that's fine then.